README.md

```diff
@@ -16,6 +16,7 @@ The project is in early alpha deep development seems-like-its-working-oh-crap st
 Supported blockchains:
 
 * [Bitcoin](https://bitcoin.org/) (WIP)
+* [Bytecoin](https://bytecoin.org/)
```

Comment:
Just a convention, every new exporter is marked work-in-progress unless tested in production.
```suggestion
* [Bytecoin](https://bytecoin.org/) (WIP)
```